haiku-apps/stopwatch/stopwatch-1.0.0.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="A simple stopwatch application"
+DESCRIPTION="This stopwatch is very simple to use and allows a user to start, stop and reset the clock."
+HOMEPAGE="https://github.com/dcatt-haiku/StopWatch"
+COPYRIGHT="2016 DCatt"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dcatt-haiku/StopWatch/archive/v1.0.0.tar.gz"
+CHECKSUM_SHA256="320f0d7502316e12209c73576cacf488f49abd58a42cc986a84882a54eb9e732"
+SOURCE_DIR="StopWatch-1.0.0"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	StopWatch = $portVersion
```

Comment:
I think I forgot to change that back as I was desperately trying anything to get the recipe to work and was taking ideas from other recipes.  Would you be willing to pull this down and try to run it and see if you can replicate the issue I'm seeing with the work-* directory?